### DISCUSSION TITLE
Introduction Thread

### DISCUSSION THREAD

**Discussion by pnbrown** [vmware-tanzu/community-edition#2102] [General]
If you're comfortable sharing, we'd love to know more about you!

Feel free to introduce yourself to the community below. We'd love to hear what you do, what brought you here, and what you're looking to get out of joining us here.

**Comment by pnbrown** [comment]
Hi everyone,
My name is Nigel and I'm the Community Manager for Tanzu Community Edition. I joined VMware to help launch this project and this community and I'm excited to be here with you all. I'm looking forward to helping build an inclusive community for learning, sharing, and contributing. If there's anything I can do to help you to that end, please don't hesitate to reach out.

**Comment by stmcginnis** [comment]
Hi all!

I'm a community open source engineer at VMware. I came here to work on supporting the Tanzu Community Edition project and its community. Really looking forward to meeting folks and learning together. Just let me know how I can help!

Sean

**Comment by jpmcb** [comment]
Howdy everyone!

My name is John - I'm an open source software engineer at VMware and I'm very excited to get started and introduce Tanzu Community Edition to everyone!

My area of expertise are in: 
- the `standalone-cluster` plugin which enables users to have a lite-weight Tanzu cluster with minimal resources.
- building `tanzu-community-edition` and it's integration with `tanzu-framework`
- I also help maintain `spf13/cobra`, a  core dependency for `tanzu`

John

**Comment by qnetter** [comment]
Howdy!

I'm Roger - my title is "Product Line Manager," but that's just fancy VMware talk for a product manager with more experience than some.

I'm responsible for maintaining the high-level project roadmap, working with PMs and engineers on other projects to help define and schedule their contributions to Tanzu Community Edition, driving agreement on our direction, and a thousand tiny little process things I think you may be happier not knowing about. :)

A big part of determining what's next is hearing what users need and want, so look for me in community meetings and office hours, the Slack channel (coming soon), and here on the Google Group.

**Comment by seemiller** [comment]
Hey! Hey!

I'm Nick, a Software Engineer with VMware. I was a long time engineer with Pivotal Tracker. My last contribution to Tracker was moving it to Kubernetes. I was amazed at how easy it was to use and how powerful it is as an application platform. I knew that I needed more. So when there was an opportunity to give back to the community by open sourcing VMware's take on Kubernetes, I joined the team.

My area of focus for Tanzu Community Edition has been packaging - creating, documenting, installing and running packages and package repositories.

**Comment by kdaula** [comment]
Hello!!
I am Kavitha. I am the engineering manager for the Tanzu Community Edition project at VMware.  Really looking forward to working together with the community! Please let me know how I can help you :)

**Comment by davidvonthenen** [comment]
Hello!

My name is David vonThenen. I'm a software engineer with VMware but a different kind of engineer. He knows what the fox says. His two cents are worth thirty-seven dollars and change. He divided by zero once and didn't blow up. He can do push-ups with no hands. Yes... that kind of different.

When David isn't off saving the world, he spends most of his time working on Tanzu Community Edition by bringing his unique skill set that deals with Kubernetes, integrating with various cloud platforms, and more recently trying to automate his way out of a job. His expertise in storage, networking, and creating memes on the fly is second to none... except when it's first to something. He has spoken on these topics at KubeCon, ContainerCon, ApacheCon, and Southern California Linux Expo (SCaLE) and the crowds went wild.
 
Seriously though, I'm super excited to be a part of the team and this community. Looking forward to helping others with their Tanzu Community Edition journey and working together to make the project even better!

**Comment by nrb** [comment]
Hey everyone! I'm Nolan Brubaker, a software engineer on Tanzu Community Edition.

I came from the Velero team here at VMware, and previously I worked on the `openstack-ansible` OpenStack distribution.

In terms of TCE, I've so far been focused on the `conformance` plugin and our build process. I'm looking forward to working with the community to make TCE a platform that's useful for everyone!

**Comment by kcoriordan** [comment]
I’m Kay, I’m the technical writer with Tanzu Community Edition. My focus is on producing user-friendly documentation. I’m really looking forward to working together with the community.

**Comment by SOUIHEL-ayoub** [comment]
Hello , this Ayoub a senior cloud eng working on VMware SDDC ,oVirt ,Microsoft Azure ,Dell EMC Storage & Servers  ..etc. I'm looking forward to working with the community .

**Comment by thefluffysysop** [comment]
Hi, I am Robert Kloosterhuis. I am Dutch, and work for ITQ, a VMware partner and integrator. 
I come from a Windows sysadmin/Storage/Infra background,  22 years in it. been working with VMware stuff for about 8 years. Have done more or less a little of everything, so have pretty broad knowledge of the VMware landscape. 

Been spending the last few years getting more comfortable with Linux/FOSS and cloud-native stuff. Since joining ITQ about 2 years ago, focusing exclusively on cloud-native, and that means Tanzu (and taking care of our ex-Pivotal customers)! 
I am becoming the go-to guy inside my company for anyone trying to understand the Tanzu portfolio generally, I like that, and try to keep informed of news when and where I can. So you will find me in all kinds of communities; vExpert, Tanzu Vanguard, VMware{code}, etc etc. 

I am more of a big-picture guy than deep-technical, yet I try focus on the various flavors of TKG in more detail and want to have tried everything hands-on if I can. I also like to speak and present publically to various types of audiences. Connecting the dots for people and pulling people across culture/technology gaps is what I love doing most. 

I love the FOSS community approach and really like how VMware is handling TCE and its other open source projects. I want to keep my ear on the ground here cause I see TCE as one of the big TKG pillars along with TKGs, TKGm and (still) TKGi.

**Comment by jbcreek** [comment]
Hello,

I am Jeff Creek.  I work for INdigital.  A next-gen 911 network and services corporation.

Our deployments are in market.  Most of them do not (and will not) have internet access.  Public cloud is not an option for most of our services.  Our applications are not cloud native.  Most of our customers require a waterfall approach to roll outs.  However, some of our developers are clamoring for Kubernetes.

I am attempting to learn enough about Kubernetes to unravel this puzzle.

**Comment by alexnaris** [comment]
Hello all!

I am Alex and I work in an open source shop providing saas services.  Have used a few different kubernetes solutions over the past years and now am extremely excited to start using tanzu and take things to the next level!
